packages/block-library/src/post-comment-avatar/editor.scss

```diff
@@ -0,0 +1,5 @@
+.wp-block-post-comment-avatar img {
```

Comment:
It seems we should be able to avoid editor-only styles for this one — what is it accomplishing?

Reply:
The ResizableBox creates a `div`, this one the one that is getting the width and height based on options selected. Then we save those variables as attributes for the frontend display.
In order to get the img inside the div to grow as the user changes those variables, we add a width and height inherit class.

We have something similar on image block: https://github.com/WordPress/gutenberg/blob/trunk/packages/block-library/src/image/editor.scss#L31-L39

We can look for another approach if needed of course!

Reply:
It's something we should consider form the perspective of the DX — adding a component like ResizableBox should not be forcing me to supply editor styles, the component itself should be handling that for me.

Reply:
I added a class to the main ResizableBox component in order for the next img to inherit the width. Let me know if that is ok. As that component is in a lot of different components of the editor :-)